src/bidiServer/mapperServer.ts

```diff
@@ -19,12 +19,14 @@ import Protocol from 'devtools-protocol';
 import WebSocket from 'ws';
 import debug from 'debug';
 
-import {CdpClient, CdpConnection, WebSocketTransport} from '../cdp/index.js';
+import {CdpClient, CdpConnection, WebSocketTransport} from '../cdp';
```

Comment:
let's use '.js' consistently and let's not import using node's resolution to be able to potentially re-use the code on the browser side.

Reply:
done

Reply:
Should this be applied to the entire codebase? I.e. always import with `.js` suffix?